modules/backend/behaviors/RelationController.php

```diff
@@ -1208,7 +1220,7 @@ public function onRelationManageDelete()
         elseif ($this->viewMode == 'single') {
             $relatedModel = $this->viewModel;
             if ($relatedModel->exists) {
-                $relatedModel->delete();
+                $relationAffected |= $relatedModel->delete();
```

Comment:
`|=`? never seen that before

Reply:
I am using this shortcut often. `True` and `false` are forced to be implemented binary (0b1, 0b0) and this is bitwise operator OR with assignment.
I can change it to cleaner logical `$relationAffected = $relationAffected || $relatedModel->delete();`